flexmeasures/ui/templates/base.html

```diff
@@ -280,13 +281,25 @@
         }
     });
 
+    function checkSourceMasking(data) {
+        var uniqueSourceIds = getUniqueValues(data, 'source.id');
+        if (chartType == 'daily_heatmap' && uniqueSourceIds.length > 1) {
+            document.getElementById('sourcewarn').style.display = 'block';
+            document.getElementById('sourcewarn').innerHTML = 'Please note that only data from the most prevalent source is shown.';
```

Comment:
If that is easy work, I'd like the source name to be displayed in the warning. But otherwise, let's go.